book/website/collaboration/leadership/leadership-story-data-librarian.md

```diff
@@ -0,0 +1,48 @@
+# Open leadership by a data librarian
+
+The following are personal reflections by The Turing Way contributor Patricia Herterich and is based on her leading a discussion session at the book dash in May 2022.
+
+## What does leadership for research infrastructure roles look like?
+
+As formal career paths for many research infrastructure roles are currently not yet defined, formal leadership positions are limited and often tied to managing people or budgets.
+In the ten years I have worked in the open science space in research infrastructure roles (research data librarian might come closest to describe it), I have not formally managed a team, though I hope that I have led and enacted change and hopefully inspired some people along the way to work more openly.
+
+*So: how do people in research infrastructure roles lead?*
+
+Many of these roles contain a degree of coordination and project management, so our leadership brings people together, facilitates conversations that otherwise might not happen and while we might not be decision makers, we certainly can get others to make decisions and hold them accountable for enacting them.
+
+I have managed a range of services (data repositories, service to support data management planning) working closely with software developers on delivering a smooth service to users.
+As I prefer decision making by consensus, it often feels more like a collaboration and less like me leading the team and service development. 
+
+
+## Inspirations
+
+Many of my leadership inspirations and ways how I acquired skills where not as part of my day to day jobs or formal certifications but communities I joined to grow my professional network.
+Usually, I lead when people hold me accountable for about 10% of the ideas that I throw out on Twitter.
```

Comment:
I'm not entirely sure what you're trying to say here: you're leading something that you said on Twitter and only when people bring it up back to you?